src/components/AddItems.js

```diff
@@ -36,6 +40,24 @@ const AddItemsToList = () => {
       .get()
       .then((data) => {
         if (data.docs.length) {
+          const { items } = data.docs[0].data();
+          const nonWordCharactersAndUnderscores = /[\W_]/g;
```

Comment:
Awesome how you used the shortened regex form!

Reply:
oh yes! This is a super clean and powerful regex! 👏